packages/studio-ui/src/components/AddPageButton/LayoutSelector.tsx

```diff
@@ -40,18 +45,24 @@ export default function LayoutSelector({
 
   const selectClasses = twMerge(selectCssClasses, "w-full mb-6");
   const body = (
-    <label>
-      <select className={selectClasses} onChange={handleChange}>
-        <option key="" value="">
-          No layout selected
+    <select
+      className={selectClasses}
+      onChange={handleChange}
+      aria-label="Layout picker"
+    >
+      <option value="" aria-label="No layout selected">
```

Comment:
I would have thought that `option` elements would automatically get labelled by their text content, so we wouldn't need to explicitly add an `aria-label`. Is that not the case?

Reply:
I thought I tried something like that earlier, but it actually does look like that works, changed!